docs(legal): add support service tos (#2234)
caos/zitadel: docs/docs/legal/terms-support-service.md

---
title: Terms and Conditions for Support Services
custom_edit_url: null
--- 
## General

### Introduction

> CAOS Ltd. (**"We"**, **CAOS AG**, or simply **CAOS**), with head office in Teufener Strasse 19, 9000 St. Gallen, Switzerland, offers amongst other services and websites (**Services**) as well commercial support services (**Support Services**) for units of CAOS software products.

Reviewer: ```suggestion
CAOS Ltd. (**"We"**, **CAOS AG**, or simply **CAOS**), with head office located at Teufener Strasse 19, 9000 St. Gallen, Switzerland, offers amongst other services and websites (**Services**) as well commercial support services (**Support Services**) for units of CAOS software products.
```

The customer relationship (**Framework Agreement** or **The Agreement**) is created by the **Customer** (**"you"**) by accepting a **Purchase Order** (**"PO"**) for the specified Support Services (**Booking**). Jointly you and CAOS will be referred to as **the Parties**. The terms of service (**"TOS"**) outlined in this document establish the most important points of this Framework Agreement – independently of the use of any other services. 

This Agreement has the following appendices. When you enter the Agreement with us, you accept these agreements.

* [**Data Processing Agreement**](data-processing-agreement) - How we process personal data on behalf of you
* [**Support Service Descriptions**](support-services) - How we provide support services to you
* [**Acceptable Use Policy**](acceptable-use-policy) - What we understand as acceptable and fair use of our Services

### Alterations

> Any provisions which deviate from these TOS must be agreed in writing between the Customer and us. Such agreements shall take precedence over the TOS outlined in this document.

Reviewer: Mail 😄

> Any provisions which deviate from these TOS must be agreed in writing between the Customer and us. Such agreements shall take precedence over the TOS outlined in this document.

Author: ```suggestion
Any provisions which deviate from these TOS must be agreed in writing (mail sufficient) between the Customer and us. Such agreements shall take precedence over the TOS outlined in this document.
```

### Transfer

You may only transfer the Framework Agreement or Services used in the context of the Framework Agreement to third parties with our prior written consent.

### Term

> Coverage under this Agreement will start with Booking of Support Services, for a minimum period of 12 months. Support Services agreements will automatically renew for additional one year terms upon submission of a purchase order for renewal, unless either you or CAOS provides written notice of termination at least 30 days prior to the end of any such term. Each renewal will be at CAOS' then-current rate. In the event that you accesses CAOS Support services in any way after the Agreement has expired or been terminated, you will continue to be bound by this Agreement, which will continue to apply to the services after such expiration or termination.

Reviewer: The mail thing 😄

### Service review

If not otherwise agreed, CAOS offers a yearly review meeting with you to discuss the service quality and any feedback you might have. We are not required to participate in the meeting after the term has expired.

### Limited influence

Be advised that the scope of our influence is limited. For example, the actual accessibility of a service is also dependent on the connection to and between various Internet Service Providers ("ISPs"). Portions of our services, i.e. software components, may also be beyond our influence and be subject to their own contractual conditions. You accept that in such cases we reject any responsibility.

### Inclusion of third parties

We may include third parties in the provision of our services. See our [Privacy Policy](privacy-policy) and our [Data Processing Agreement](data-processing-agreement) for more information.

## Your obligations

### Contact information

At our request you will provide your truthful contact information and keep it updated at all times. You must also ensure that you actually receive messages, in particular emails, intended for you.

### Use

> You will ensure that the use of our Websites and Services by you or third parties complies with all applicable legislation, these these TOS, and our [Acceptable Use Policy](acceptable-use-policy) at all times.

Reviewer: ```suggestion
You will ensure that the use of our Websites and Services by you or third parties complies with all applicable legislation, these TOS, and our [Acceptable Use Policy](acceptable-use-policy) at all times.
```

### Support Process

You will ensure to follow the support process, especially provide all required initial information to the issue, as outlined in the [Annex](support-services) to this document.

### Training of support staff

You will ensure regular training of your support staff. Your support staff must be able to provide the required information for support issues to us, and thus requires access and up-to-date knowledge of the services.

Initial know-how transfer for the services will be organized in training sessions conducted by us. We can provide knowledge sessions throughout the term to train newly onboarded staff, update your support staff about important updates, or refresh knowledge in specified areas. In case we notice insufficient quality of support requests from Customers, we will propose appropriate training sessions.

### Security

You will take appropriate measures to prevent any misuse of the services you booked. These include, for example, securing the software used and the prompt installation of security updates as well as using suitably secure passwords.

### Disaster recovery

> We take care of the necessary disaster recovery measures. The goal is to maintain a maximum 24h old restore point off all the vital data.

Reviewer: we don't provide this with supported units, only dedicated

> We take care of the necessary disaster recovery measures. The goal is to maintain a maximum 24h old restore point off all the vital data.

Author: ```suggestion
```

If the maintenance of service quality requires your cooperation, for example to remedy errors in the services you use, you will provide said cooperation promptly and free of charge.

### Third party obligations

You will ensure that your vicarious agents, customers and third parties fulfill these obligations as well.

## Financial

### Payment

Fees for the initial or any subsequent term of Support Services booked will be due and payable net 30 days from date of invoice. All payments to CAOS will be made in CHF or EUR to our bank account, as indicated in the PO.

### Offsetting

Offsetting against a counterclaim is prohibited.

### Collection

In the event of default we reserve the right to transfer our claim to a collections agency. You will bear any resulting costs insofar as legally permissible.

### Lapsed Service Fee

In case the term of the Support Service contract has expired within 1 to 180 days, you will be required to pay a Lapsed Service fee in addition to purchasing and activating a one-year renewal contract term at the then-current fee and conditions. The renewal term's start date will also be backdated to begin coverage from the service's original expiration date.

> Please contact us for current fees.

Reviewer: ```suggestion
Please contact us for the current fees.
```

Author: 'the' is not necessary here, as we speak of the general fees. We could say 'contact us for the current lapsed service fee'.

### Recertification Fee

Recertification of a unit, to be covered under Support Services, is required for:

* units for which Support Services have been expired for more than 180 days

> * units that run a release that is older than 180 days from the products most recent release

Reviewer: ```suggestion
* units that run a release that is older than 180 days from the products most recent stable release
```

* requests for support on products and services purchased or supported from non-authorized resellers

> Recertification of a unit requires payment of a Recertification Fee which results in a [checkup](https://docs.zitadel.ch/docs/guides/installation/checkup) of the unit by CAOS. The unit will be inspected to asses its condition and eligibility for service coverage.

Reviewer: Please use a softlink

> Recertification of a unit requires payment of a Recertification Fee which results in a [checkup](https://docs.zitadel.ch/docs/guides/installation/checkup) of the unit by CAOS. The unit will be inspected to asses its condition and eligibility for service coverage.

Author: ```suggestion
Recertification of a unit requires payment of a Recertification Fee which results in a [checkup](/docs/guides/installation/checkup) of the unit by CAOS. The unit will be inspected to asses its condition and eligibility for service coverage.
```
